src/cart/crt_iv.c

```diff
@@ -2062,7 +2062,8 @@ handle_ivsync_response(const struct crt_cb_info *cb_info)
 	struct iv_sync_cb_info	*iv_sync = cb_info->cci_arg;
 	struct crt_iv_ops	*iv_ops;
 
-	crt_bulk_free(iv_sync->isc_bulk_hdl);
+	if (iv_sync->isc_bulk_hdl != CRT_BULK_NULL)
+		crt_bulk_free(iv_sync->isc_bulk_hdl);
 
```

Comment:
At this point, isn't it wise to set iv_sync->isc_bulk_hdl to CRT_BULK_NULL just to make sure no other code tries to use a free structure/memory?

Reply:
This is the end of life for iv_sync. it will get FREE-ed on line 2098 in this function so no real point of setting this to null at this point